File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_07289.py
# SPDX-License-Identifier: Apache-2.0
from flask import request


request_state: dict[str, str] = {}

def BenchmarkTest07289():
    raw_body = request.get_data(as_text=True)
    request_state['last_input'] = raw_body
    data = request_state['last_input']
    return str(data), 200, {'Content-Type': 'text/html'}

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest07289.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest07289 {

    @PostMapping(path="/BenchmarkTest07289", consumes="text/plain")
    public void BenchmarkTest07289(@RequestBody String rawBody, HttpServletRequest request, HttpServletResponse response) throws Exception {
        response.sendError(500, "Internal error");
    }
}
